### PR TITLE
Add documentation for API events

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/common/schemas/output/ledger-closed.json [hunk 1]
```diff
@@ -4,15 +4,48 @@
...
-    "ledgerVersion": {"$ref": "ledgerVersion"},
-    "ledgerTimestamp": {"type": "string", "format": "date-time"},
-    "reserveBase": {"type": "integer", "minimum": 0},
-    "reserveIncrement": {"type": "integer", "minimum": 0},
-    "transactionCount": {"type": "integer", "minimum": 0},
-    "validatedLedgerVersions": {"type": "string"}
+    "feeBase": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Base fee, in drops."
```
Comment (mDuo13) [inline src/common/schemas/output/ledger-closed.json]: Does RippleAPI actually explain the concept of Drops anywhere else? This will be confusing otherwise.

File: src/common/schemas/output/ledger-closed.json [hunk 2]
```diff
@@ -4,15 +4,48 @@
...
+    },
+    "feeReference": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Cost of the 'reference transaction' in 'fee units'."
```
Comment (mDuo13) [inline src/common/schemas/output/ledger-closed.json]: This is also confusing, although sadly, I don't think we have a good explanation that's not overly complicated (because the product itself is just complicated here). Maybe not worth exposing in RippleAPI?

**Comment by mDuo13** [comment]
Left some minor quibbles, but overall LGTM.

File: src/common/schemas/output/ledger-closed.json (hunk 1 above)
Comment (clark800) [inline src/common/schemas/output/ledger-closed.json]: I have to fix this to make it return XRP, but since it's a breaking change, I was going to leave it for a separate commit.

File: src/common/schemas/output/ledger-closed.json (hunk 2 above)
Comment (clark800) [inline src/common/schemas/output/ledger-closed.json]: It would be nice if we could just link to a separate page that explains fees in more detail.
